vpn.py

```diff
@@ -95,16 +110,20 @@ def scan_vpns(directory, auth_file, exclude_list):
         except Exception as exp:
             logging.error("%s: Failed to set VPN info: %s" % (filename, exp))
 
-        if not centinel.backend.experiments_available(config.params):
-            logging.info("%s: No experiments available." % (filename))
-            continue
-
         logging.info("%s: Synchronizing." % (filename))
         try:
             centinel.backend.sync(config.params)
         except Exception as exp:
             logging.error("%s: Failed to sync: %s" % (filename, exp))
 
+        if not centinel.backend.experiments_available(config.params):
+            logging.info("%s: No experiments available." % (filename))
+            try:
+                centinel.backend.set_vpn_info(config.params, vpn_address, country)
```

Comment:
To make sure client IP address for client doesn't change on the server (`experiments_available()` contacts the server, which changes its IP address).
I will make the server treat VPN IP addresses like their country so that multiple calls to this function can be avoided.